main.js

```diff
@@ -0,0 +1,68 @@
+import Contextmenu from './src/javascript/contextmenu.js';
+import Contentview from './src/javascript/contentview.js';
+import Item from './src/javascript/item.js';
+
+const init = () => {
+  const itemTarget = document.getElementById('itemTarget');
+  const searchInput = document.getElementById('searchInput');
+  const contentview = document.getElementById('contentview');
+  const saveButton = document.getElementById('saveButton');
+  const contextMenu = new Contextmenu(itemTarget);
+  const contentView = new Contentview(contentview);
+  
+  let cache = getData();
+  if(!cache) cache = setData();
```

Comment:
p3;
[Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)(`??`) 을 쓰면 좀 더 깔끔하게 표현이 가능하겟네요 

보통 default 값 세팅할 때 `||` 논리 연산자로 많이들 했는데, 이제 nullish coalescing 도 쓸 수 있게 됬어요


```suggestion
// const cache = getData() || setData(); 
  const cache = getData() ?? setData();
```

Reply:
> p3;
> [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)(`??`) 을 쓰면 좀 더 깔끔하게 표현이 가능하겟네요
> 
> 보통 default 값 세팅할 때 `||` 논리 연산자로 많이들 했는데, 이제 nullish coalescing 도 쓸 수 있게 됬어요

따끈따끈한 연산자군요! 좋은 정보 감사드립니다!
적어주신 대로 get에서 논리 연산으로 필터링 가능한 값을 적어주는데 분기를 만들 이유가 없었네요! ::)